registry/insecure.md

```diff
@@ -26,7 +26,15 @@ environment.
 
     Depending on your operating system, your Engine daemon start options.
 
-2. Edit (or add) the `DOCKER_OPTS` line and add the `--insecure-registry` flag.
+2. **If using docker version 1.12 or greater**
+
+    Create or modify /etc/docker/daemon.json on the client machine and add:
+     
+    `{ "insecure-registries":["myregistry.example.com:5000"] }`
+
```

Comment:
Would look better if this line were encased in a backtick fence using three backticks, like so:

```
{ "insecure-registries":["myregistry.example.com:5000"] }
```

Wrapping it in single backticks is just for inline use. I think you're trying to show a snippet here.